docs/circuit_cutting/tutorials/index.rst

```diff
@@ -1,13 +1,9 @@
+#########################
+Circuit Cutting Tutorials
+#########################
 .. include:: README.rst
```

Comment:
The `README.rst` in this directory should be updated, too.  For an example of what it should look like, see `docs/circuit_cutting/cutqc/tutorials/README.rst`.